package.json

```diff
@@ -106,7 +106,7 @@
   "dependencies": {
     "@opentelemetry/api": "^1.7.0",
     "@opentelemetry/core": "1.20.0",
-    "@opentelemetry/instrumentation-xml-http-request": "^0.41.0",
+    "@opentelemetry/instrumentation-xml-http-request": "^0.51.1",
```

Comment:
upgrading as 0.41.0 imports `@opentelemetry/api@1.4.1` and it throws an error due to multiple api versions